core/src/main/kotlin/com/alecstrong/sql/psi/core/SqlFileBase.kt

```diff
@@ -37,12 +39,24 @@ abstract class SqlFileBase(
     return schema(T::class, sqlStmtElement, includeAll)
   }
 
+  private val systemTables: List<TableElement> by lazy {
+    val psiFactory = PsiFileFactory.getInstance(project)
+    predefinedTables.flatMap { predefinedTable ->
+      val sqlFile = psiFactory.createFileFromText(predefinedTable.fileName, language, predefinedTable.content) as SqlFileBase
+      val stmts = sqlFile.sqlStmtList!!.stmtList
+      stmts.mapNotNull { it.firstChild as? TableElement }
```

Comment:
Currently, this function only supports TableElements. In theory, we could support all valid content, but the schema is typed, so we would need to iterate over the content of the file.

Reply:
Fixed.